CYCLONEDDS_QUALITY_DECLARATION.md

```diff
@@ -140,11 +180,16 @@ If `CycloneDDS` is built without security enabled, the product has no apparent r
 
 ## Platform Support [6]
 
-The `CycloneDDS` [repository documentation](https://github.com/eclipse-cyclonedds/cyclonedds/blob/master/README.md#building-eclipse-cyclone-dds) states that Linux, Mac and Windows 10 are supported.
-Platform version requirements are not explicitly stated, nor are all of the minimum build tool versions.
```

Comment:
Basically, the minimum versions are not known exactly. For Linux and macOS, ancient versions will still do the trick. For evidence, I suppose it being possible to build on [Solaris 2.6 on SPARCv8](https://github.com/eclipse-cyclonedds/cyclonedds/tree/master/ports/solaris2.6) (given pre-generated header files and IDL output) is a fair indication.

CMake 3.7, Java 1.8 and Maven 3.5 are new enough.

So while this statement is true, it is a disingenious one. (@joespeed I'm aware of the fact that you chose to delete it.)

Reply:
correct.